### PR TITLE
Added the blogs and Youtube links

### PR DESCRIPTION
**Pull request by mnik7044**
Worked on the issue #10 I have added the youtube videos and blogs for the helm and prometheus file, also added the argocd blog and corrected the format in the rust file.

### CODE REVIEW

**Comment by mnik7044** [comment]
Hey @kaiwalyakoparkar  have you reviewed it?

**Comment by kaiwalyakoparkar** [comment]
Hey @mnik7044 thank you so much for your patience, I will start reviewing it shortly

File: Helm/Readme.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+# Helm
+You can learn helm from the given Blog and Youtube Videos
+- [Blogs](https://blogs.kaiwalyakoparkar.com/what-is-helm)
+
+[![YouTube Video](https://img.youtube.com/vi/fw97GL6LHhg/hqdefault.jpg)](https://youtu.be/fw97GL6LHhg) 
+
```
Comment (kaiwalyakoparkar) [inline Helm/Readme.md:6]: Kindly remove such extra lines

File: Helm/Readme.md [hunk 2]
```diff
@@ -31,4 +40,7 @@ helm repo update <chart-name>
 
 # search a keywork in chart
 helm search repo <keyword>
-```
\ No newline at end of file
+```
+
```
Comment (kaiwalyakoparkar) [inline Helm/Readme.md:44]: Same. remove unnecessary lines here

File: Prometheus/Readme.md [hunk 3]
```diff
@@ -1,4 +1,12 @@
-#Steps and commands
+# Prometheus
+You can learn Prometheus from the following blogs and youtube videos
+
+- [Blog](https://blogs.kaiwalyakoparkar.com/introduction-to-prometheus)
+
+  [![YouTube Video](https://img.youtube.com/vi/EcixeZzyaVc/hqdefault.jpg)](https://youtu.be/EcixeZzyaVc)
+
+
```
Comment (kaiwalyakoparkar) [inline Prometheus/Readme.md:8]: Remove this extra lines

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

**Comment by mnik7044** [comment]
Hey @kaiwalyakoparkar  just wanted to know that whether I need to delete this pr and add a new pr or if I do the changes now it will be reflected in the current pr only?

**Comment by mnik7044** [comment]
Hey @kaiwalyakoparkar I have removed all the extra lines, please review it.

File: Prometheus/Readme.md [hunk 4]
```diff
@@ -1,5 +1,8 @@
-#Steps and commands
-
+# Prometheus
+You can learn Prometheus from the following blogs and youtube videos
+- [Blog](https://blogs.kaiwalyakoparkar.com/introduction-to-prometheus)
+ [![YouTube Video](https://img.youtube.com/vi/EcixeZzyaVc/hqdefault.jpg)](https://youtu.be/EcixeZzyaVc)
+# Steps and commands
```
Comment (kaiwalyakoparkar) [inline Prometheus/Readme.md:5]: Please add one empty line before and after the heading please (for all the files)

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

**Review by kaiwalyakoparkar — APPROVED** [review]
lgtm, thank you so much for contributing

**Comment by mnik7044** [comment]
Thank you for this opportunity will be looking to contribute more in the coming days.

**Pull request merged**